pkger/parser.go

```diff
@@ -594,9 +594,12 @@ func (p *Pkg) buckets(hideRemoved bool) []*bucket {
 	return buckets
 }
 
-func (p *Pkg) checks() []*check {
+func (p *Pkg) checks(hideRemoved bool) []*check {
```

Comment:
Nitpicky comment, but I typically prefer separate functions over boolean arguments (https://martinfowler.com/bliki/FlagArgument.html) to improve readability and detangle logic.

Reply:
good call. Glad you pointed this out, definitely unneccessary. I just changed it to return all as it was initially doing. I do the filtering at callsite instead.